data/vivek839.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Vivek nath",
+  "type": "personal",
+  "bio": "The java code , OpenSource,remote work ,Linux, every one to learn code  because it develop you logic power",
+  "links": [
+  {
+  "name": "Follow me on twitter",
+  "url": "https://twitter.com/Viveknath",
+  "icon": "FaTwitter"
+},
+{
+  "name": "Follow me on Github",
+  "url": "https://github.com/vivek839",
+  "icon": "GithubAlt"
```

Comment:
there is no icon name with `GithubAlt`, for `GitHub` icon you can use `FaGithub`. For more icon please visit [here](https://linkfree.eddiehub.io/icons)